examples/dropna_example.py

```diff
@@ -0,0 +1,102 @@
+import os
```

Comment:
Thanks for sharing this - I'm glad to know it works in the larger workflow. 

I don't think we need to maintain this as a separate example though, can you remove?